### PR TITLE
dev-java/jctools-core: new package

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: dev-java/jctools-core/jctools-core-2.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# POM: pom.xml
+# test? junit:junit:4.12 -> >=dev-java/junit-4.13.2:4
+# test? org.hamcrest:hamcrest-all:1.3 -> !!!artifactId-not-found!!!
+
+DEPEND=">=virtual/jdk-1.8:*
+	test? ( dev-java/hamcrest-library:1.3 )"
+
+RDEPEND=">=virtual/jre-1.8:*"
+
+DOCS=(
```
Comment (Flowdalic) [inline dev-java/jctools-core/jctools-core-2.0.2.ebuild:33]: pkgcheck reports a whitespace error here

File: dev-java/jctools-core/jctools-core-2.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="org.jctools:jctools-core:2.1"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Java Concurrency Tools Core Library"
+HOMEPAGE="http://jctools.github.io/JCTools/"
```
Comment (Flowdalic) [inline dev-java/jctools-core/jctools-core-2.0.2.ebuild:16]: pkgcheck reports that https is available for this url

File: dev-java/jboss-jms-api/jboss-jms-api-1.0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="org.jboss.spec.javax.jms:jboss-jms-api_1.1_spec:1.0.1.Final"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="JSR-000914: Java(TM) Message Service (JMS) 1.1 API"
+HOMEPAGE="http://www.jboss.org/jboss-jms-api_1.1_spec"
```
Comment (Flowdalic) [inline dev-java/jboss-jms-api/jboss-jms-api-1.0.1.ebuild:15]: pkgcheck reports that this is a dead URL

**Review by Flowdalic — CHANGES_REQUESTED** [review]

File: dev-java/jctools-core/jctools-core-2.0.2.ebuild (hunk 1 above)
Comment (vaukai) [reply dev-java/jctools-core/jctools-core-2.0.2.ebuild]: Done.

File: dev-java/jctools-core/jctools-core-2.0.2.ebuild (hunk 2 above)
Comment (vaukai) [reply dev-java/jctools-core/jctools-core-2.0.2.ebuild]: Done.

File: dev-java/jboss-jms-api/jboss-jms-api-1.0.1.ebuild (hunk 3 above)
Comment (vaukai) [reply dev-java/jboss-jms-api/jboss-jms-api-1.0.1.ebuild]: Replaced.

**Comment by Flowdalic** [comment]
Seems like this was closed in favor of https://github.com/gentoo/gentoo/pull/24106. Is this correct?

**Comment by vaukai** [comment]
> Seems like this was closed in favor of #24106. Am I right?

Correct.  Having `dev-java/kafka-clients` in the tree made it possible.
Sorry for causing confusion

**Pull request closed without merging**